

30-08-2011 16:12:37
Error-->>TallySynchronizer.TallyException: Error On Modification
		


30-08-2011 16:12:42
Error-->>TallySynchronizer.TallyException: Ledger Not Found
		

Request 
<ENVELOPE>
<HEADER>
<VERSION>1</VERSION>
<TALLYREQUEST>Import</TALLYREQUEST>
<TYPE>Data</TYPE>
<ID>All Masters</ID>
</HEADER>
<BODY>
<DESC>
<STATICVARIABLES>
<SVCURRENTCOMPANY>Tally_ICMS</SVCURRENTCOMPANY>
<IMPORTDUPS>@@DupIgnoreCombine</IMPORTDUPS>
</STATICVARIABLES>
</DESC>
<DATA>
<TALLYMESSAGE>
<VOUCHER DATE="20110830" TAGNAME = "Reference" TAGVALUE="465630CHR" Action="Alter">
<DATE>20110830</DATE>
<NARRATION>[465630CHR][13.8] From : ram(r111)Desc : s</NARRATION>
<VOUCHERTYPENAME>Journal</VOUCHERTYPENAME>
<VOUCHERNUMBER>465630CHR</VOUCHERNUMBER>
<REFERENCE>465630CHR</REFERENCE>
<ALLLEDGERENTRIES.LIST>
<LEDGERNAME>GYM CHARGES</LEDGERNAME>
<ISDEEMEDPOSITIVE>No</ISDEEMEDPOSITIVE>
<AMOUNT>200</AMOUNT>
</ALLLEDGERENTRIES.LIST>
<ALLLEDGERENTRIES.LIST>
<LEDGERNAME>SMART CARD ACCOUNT</LEDGERNAME>
<ISDEEMEDPOSITIVE>Yes</ISDEEMEDPOSITIVE>
<AMOUNT>-200</AMOUNT>
</ALLLEDGERENTRIES.LIST>

</VOUCHER>
</TALLYMESSAGE>
</DATA>
</BODY>
</ENVELOPE>

Response 
<ENVELOPE>
 <HEADER>
  <VERSION>1</VERSION>
  <STATUS>1</STATUS>
 </HEADER>
 <BODY>
  <DATA>
   <IMPORTRESULT>
    <LINEERROR>   34: Ledger &apos;GYM CHARGES&apos; does not exist!(FOR OBJECT: &apos;Date:&quot;20110830&quot;:Reference:&quot;465630CHR&quot;&apos;)    </LINEERROR>
    <CREATED>0</CREATED>
    <ALTERED>0</ALTERED>
    <LASTVCHID>0</LASTVCHID>
    <LASTMID>0</LASTMID>
    <COMBINED>0</COMBINED>
    <IGNORED>0</IGNORED>
    <ERRORS>1</ERRORS>
   </IMPORTRESULT>
  </DATA>
 </BODY>
</ENVELOPE>



30-08-2011 16:12:43
Error-->>TallySynchronizer.TallyException: <ENVELOPE>
 <HEADER>
  <VERSION>1</VERSION>
  <STATUS>1</STATUS>
 </HEADER>
 <BODY>
  <DATA>
   <IMPORTRESULT>
    <LINEERROR>    1: Ledger &apos;SMART CARD ACCOUNT&apos; already exists!(FOR OBJECT: &apos;SMART CARD ACCOUNT(003)&apos;)    </LINEERROR>
    <CREATED>0</CREATED>
    <ALTERED>0</ALTERED>
    <LASTVCHID>0</LASTVCHID>
    <LASTMID>0</LASTMID>
    <COMBINED>0</COMBINED>
    <IGNORED>0</IGNORED>
    <ERRORS>1</ERRORS>
   </IMPORTRESULT>
  </DATA>
 </BODY>
</ENVELOPE>

Response
<ENVELOPE><HEADER><VERSION>1</VERSION><TALLYREQUEST>Import</TALLYREQUEST><TYPE>Data</TYPE><ID>All Masters</ID></HEADER><BODY><DESC><STATICVARIABLES><SVCURRENTCOMPANY>Tally_ICMS</SVCURRENTCOMPANY><IMPORTDUPS>@@DupIgnoreCombine</IMPORTDUPS></STATICVARIABLES></DESC><DATA><TALLYMESSAGE><LEDGER NAME="SMART CARD ACCOUNT(003)"><PARENT>Direct Incomes</PARENT><LANGUAGENAME.LIST><NAME.LIST TYPE="String"><NAME>SMART CARD ACCOUNT(003)</NAME><NAME>SMART CARD ACCOUNT</NAME></NAME.LIST></LANGUAGENAME.LIST></LEDGER></TALLYMESSAGE></DATA></BODY></ENVELOPE>
		Ledger Name : SMART CARD ACCOUNT(003)
Alias Name : SMART CARD ACCOUNT

 Request XML 
<ENVELOPE><HEADER><VERSION>1</VERSION><TALLYREQUEST>Import</TALLYREQUEST><TYPE>Data</TYPE><ID>All Masters</ID></HEADER><BODY><DESC><STATICVARIABLES><SVCURRENTCOMPANY>Tally_ICMS</SVCURRENTCOMPANY><IMPORTDUPS>@@DupIgnoreCombine</IMPORTDUPS></STATICVARIABLES></DESC><DATA><TALLYMESSAGE><LEDGER NAME="SMART CARD ACCOUNT(003)"><PARENT>Direct Incomes</PARENT><LANGUAGENAME.LIST><NAME.LIST TYPE="String"><NAME>SMART CARD ACCOUNT(003)</NAME><NAME>SMART CARD ACCOUNT</NAME></NAME.LIST></LANGUAGENAME.LIST></LEDGER></TALLYMESSAGE></DATA></BODY></ENVELOPE>

 Response XML 
<ENVELOPE>
 <HEADER>
  <VERSION>1</VERSION>
  <STATUS>1</STATUS>
 </HEADER>
 <BODY>
  <DATA>
   <IMPORTRESULT>
    <LINEERROR>    1: Ledger &apos;SMART CARD ACCOUNT&apos; already exists!(FOR OBJECT: &apos;SMART CARD ACCOUNT(003)&apos;)    </LINEERROR>
    <CREATED>0</CREATED>
    <ALTERED>0</ALTERED>
    <LASTVCHID>0</LASTVCHID>
    <LASTMID>0</LASTMID>
    <COMBINED>0</COMBINED>
    <IGNORED>0</IGNORED>
    <ERRORS>1</ERRORS>
   </IMPORTRESULT>
  </DATA>
 </BODY>
</ENVELOPE>





30-08-2011 16:13:12
Error-->>TallySynchronizer.TallyException: Error On Modification
		


30-08-2011 16:13:15
Error-->>TallySynchronizer.TallyException: Error On Modification
		


30-08-2011 17:42:30
Error-->>System.Threading.AbandonedMutexException: The wait completed due to an abandoned mutex.
   at System.Threading.WaitHandle.WaitOne(Int64 timeout, Boolean exitContext)
   at System.Threading.WaitHandle.WaitOne(Int32 millisecondsTimeout, Boolean exitContext)
   at System.Threading.WaitHandle.WaitOne(Int32 millisecondsTimeout)
   at TallySynchronizer.Tally.ExportOffLineTransactions() in D:\TEST\TallySynchronizer\TallyManager.cs:line 122
		
